gramps/gen/utils/place.py

```diff
@@ -46,11 +47,11 @@
 # -------------------------------------------------------------------------
 
 # translation of N/S/E/W, make sure translator understands
-degrees = "1"
-North = _("%(north_latitude)s N") % {"north_latitude": degrees}
-South = _("%(south_latitude)s S") % {"south_latitude": degrees}
-East = _("%(east_longitude)s E") % {"east_longitude": degrees}
-West = _("%(west_longitude)s W") % {"west_longitude": degrees}
+degrees: str = "1"
+North: str = _("%(north_latitude)s N") % {"north_latitude": degrees}
+South: str = _("%(south_latitude)s S") % {"south_latitude": degrees}
+East: str = _("%(east_longitude)s E") % {"east_longitude": degrees}
+West: str = _("%(west_longitude)s W") % {"west_longitude": degrees}
```

Comment:
This seems to be induced by the `_` function; would it possible to annotate that instead?

Reply:
I've added a return type hint to gettext and removed these explicit types. Thanks